### ISSUE TITLE
Feature request: template-deployment

### ISSUE THREAD

**Issue by lnfernux** [zolderio/misp-to-sentinel#1]
Hey. 

Great project - was just wondering if there was a possibility to simplify deployment and create an ARM/Bicep template for deploying to Azure?

https://github.com/Azure/azure-quickstart-templates/tree/master/100-blank-template

**Comment by wez3** [comment]
Agree, would be a nice feature. Currently no priority for me, but if anyone else wants to contribute this feature, more than welcome
